README.md

```diff
@@ -16,16 +16,58 @@ The Filecoin box has the following requirements:
 
 ## Installation
 
+Installation takes place in three parts:
+ - [Installing the filecoin box](#installing-the-filecoin-box),
+ - (optionally) [Installing the Filecoin Network Explorer](#installing-the-filecoin-network-explorer), and
+ - [Installing the front-end gallery](#installing-the-front-end-gallery)
+
+#### Installing the Filecoin Box
+
+In a terminal window, start off by installing this box.
+
 ```bash
 $ truffle unbox filecoin
 $ npm install
```

Comment:
Nit: Not something you did, but do you mind removing the "$ " in front of all the commands in this?  That will let users more easily copy/paste commands without deleting those characters from every line.  Example select:
![image](https://user-images.githubusercontent.com/1020682/146429968-6762b541-a1a2-4797-af4c-84600929cfe6.png)

Reply:
agree, fixed!